### ISSUE TITLE
Module suspended because Zygisk is not enabled

### ISSUE THREAD

**Issue by XdekHckr** [topjohnwu/Magisk#7623]
So because of that I can't use any module or do anything with root. If happened after updating to latest beta update (it was working with older beta update of android 14). Doesn't matter what version of magisk I use beta, stable or debug, zygisk just doesn't work. Android developers made something that broke zygisk feature I think. Any idea?
@
![Screenshot_20231213-161029](https://github.com/topjohnwu/Magisk/assets/43490312/75d9eb4d-734d-4be2-8ec1-8fbb34f95d9b)
![Screenshot_20231213-161035](https://github.com/topjohnwu/Magisk/assets/43490312/2544ba39-ddfc-44b8-a747-fb5042574929)

Device:
Android version: android 14 beta (build number: AP11.231117.006), PIXEL FOLD
Magisk version name: debug
Magisk version code: 26402

**Comment by yujincheng08** [comment]
Duplicate of #7619

**Comment by XdekHckr** [comment]
> Duplicate of #7619

yeah, after compiling magisk by myself I found out that zygisk is loading properly now, it's pretty sad that nobody updated apks so I had to compile it by myself. In case someone wants my build here it is (original, not modified by me, from official magisk repo, but compiled latest version): https://github.com/XdekHckr/Magisk/releases/tag/release
If someone don't trust me then feel free to use other fork, Magisk by @HuskyDG that is named Kitsune Mask (version code: 26402, version: R6578883F-kitsune). He managed to make it even better and more realiable for beta updates because of:
[Zygisk] Changed the method of injecting zygote process by using ptrace implementation from [ZygiskNext](https://github.com/Dr-TSNG/ZygiskNext)
[Zygisk] Added support for GrepheneOS Android 14, a privacy and security focused mobile OS with Android app compatibility. This is because the official Magisk does not support it.
and more!
